ONSdigital/eq-smoke-tests
# Check previewed questionnaire all the way through submission.

This tests [Runner Issue 1442](https://github.com/ONSdigital/eq-survey-runner/issues/1442)
Should fail currently and then fixed by [Runner PR 1450](https://github.com/ONSdigital/eq-survey-runner/pull/1450)
Dependent on the above PR being merged first.

To test:
Run with Runner on master and test should fail.
Run with Runner on branch **fix-issue-1442** and it should pass.
